src/main/java/org/spongepowered/api/CatalogTypes.java

```diff
@@ -103,6 +106,10 @@
 
     // SORTFIELDS:ON
 
+    public static final Class<AccountDeletionResult> ACCOUNT_DELETION_RESULT = AccountDeletionResult.class;
+
+    public static final Class<AccountType> ACCOUNT_TYPE = AccountType.class;
+
```

Comment:
I think you need to add TypeTokens for all of these new Catalog Types (unless there is some other reason I don't know of)

Reply:
Why? They're not accessible through the data api...

Reply:
Well I guess the second half of my question applies then, just didn't know.